src/context.rs

```diff
@@ -150,9 +151,13 @@ impl NativeContext {
         ])
         .add_regions([module_region])
         .build()?;
-        assert!(op.verify(), "module operation is not valid");
 
-        let mut module = Module::from_operation(op).expect("module failed to create");
+        if !op.verify() {
+            native_panic!("module operation is not valid");
+        }
```

Comment:
Ideally we'd have a `native_assert` macro.

Reply:
Done!